Makefile

```diff
@@ -45,6 +47,63 @@ openshift-deploy-kiali-test-breadth-sink:
 	@echo About to deploy Kiali Test Breadth Sink to OpenShift
 	ansible-playbook ${PLAYBOOK} -e deployment_type=${DEPLOYMENT_TYPE} -e number_of_apps=${NUM_APPS} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-breadth-sink"]}' -v
 
-openshift-deploy-kiali-complex-mesh:
-	@echo About to deploy the Kiali Complex Test Mesh to OpenShift
-	ansible-playbook ./ansible/deploy_complex_test_mesh.yml -v
\ No newline at end of file
+
+operator-build:
+	@echo Building operator
+	cd operator/kiali-test-mesh-operator && operator-sdk build ${OPERATOR_IMAGE}
+	docker push ${OPERATOR_IMAGE}
+
+
+deploy-bookinfo-manual-sidecar:
+	@echo Deploy Bookinfo with Manual Injection of the sidecar on Openshift
+	oc create -f operator/kiali-test-mesh-operator/deploy/cr/manual-sidecar/bookinfo-cr.yaml
+
+deploy-bookinfo-automatic-sidecar:
+	@echo Deploy Bookinfo with Automatic Injection of the sidecar on Openshift
+	oc create -f operator/kiali-test-mesh-operator/deploy/cr/automatic-sidecar/bookinfo-cr.yaml
+
+deploy-complex-mesh-manual-sidecar:
+	@echo Deploy Complex Mesh with Manual Injection of the sidecar on Openshift
+	oc create -f operator/kiali-test-mesh-operator/deploy/cr/manual-sidecar/complex_mesh-cr.yaml
+
+deploy-complex-mesh-automatic-sidecar:
+	@echo Deploy Complex Mesh with Automatic Injection of the sidecar on Openshift
+	oc create -f operator/kiali-test-mesh-operator/deploy/cr/automatic-sidecar/complex_mesh-cr.yaml
+
+
+remove-bookinfo-manual-sidecar:
+	@echo Deploy Bookinfo with Manual Injection of the sidecar on Openshift
+	oc delete -f operator/kiali-test-mesh-operator/deploy/cr/manual-sidecar/bookinfo-cr.yaml
+
+remove-bookinfo-automatic-sidecar:
+	@echo Deploy Bookinfo with Automatic Injection of the sidecar on Openshift
+	oc delete -f operator/kiali-test-mesh-operator/deploy/cr/automatic-sidecar/bookinfo-cr.yaml
+
+remove-complex-mesh-manual-sidecar:
+	@echo Deploy Complex Mesh with Manual Injection of the sidecar on Openshift
+	oc delete -f operator/kiali-test-mesh-operator/deploy/cr/manual-sidecar/complex_mesh-cr.yaml
+
+remove-complex-mesh-automatic-sidecar:
+	@echo Deploy Complex Mesh with Automatic Injection of the sidecar on Openshift
+	oc delete -f operator/kiali-test-mesh-operator/deploy/cr/automatic-sidecar/complex_mesh-cr.yaml
+
+
+
+operator-deploy-openshift: operator-remove-openshift
+	@echo Deploy Kiali Tesh Mesh Operator on Openshift
+	oc new-project kiali-test-mesh-operator
+	oc create -f operator/kiali-test-mesh-operator/deploy/bookinfo-crd.yaml
```

Comment:
notice here because of the "new-project", the default project will be set to "kiali-test-mesh-operator". All the CRs will be created in that namespace (because all of these oc create cmds do not override the default via -n).

OK, but now look at the remove- targets - they all have "oc delete" without specifying -n, but they also do not specify "oc project kiali-test-mesh-operator" - so the remove may not remove successfully if the user has set the default namespace to something else.

All of this is to say - if it is safe to put all of these CRs in kiali-test-mesh-operator namespace, then all the deploy- and remove- targets should explicitly specify a `-n kiali-test-mesh-operator` option to explicitly make sure you put these (and remove them from) the correct namespace.